presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java

```diff
@@ -101,6 +101,14 @@ private static void verifyJavaVersion()
             return;
         }
 
+        if (!Boolean.getBoolean("presto-temporarily-allow-java8")) {
+            failRequirement("" +
+                    "Future versions of Presto will require Java 11 starting in April 2020.\n\n" +
```

Comment:
I think it should be like 6 months from now.

Reply:
This date is the one announced previously on Slack. https://prestosql.slack.com/archives/CFLB9AMBN/p1580870213234300

However, the code build today will continue to work and we will unlikely do a release on April 1st, so I'd use a different wording here

```suggestion
                    "Future versions of Presto will require Java 11 after March 2020.\n\n" +
```